certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -2461,6 +2461,315 @@
           }
         ]
       }
+    },
```

Comment:
@ecardakaris-knowledgeonecorp Please remove this file from your PR as it belongs to another connector.

Reply:
Ok removed

Reply:
@ecardakaris-knowledgeonecorp it is still exists in your PR